packages/eslint-plugin-react/index.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   configs: {
     default: {
-      extends: ['@m6web', 'prettier', 'prettier/react'],
+      extends: ['airbnb', 'prettier', 'prettier/react', '@m6web'],
```

Comment:
Changing this order breaks 6play linting ⚠️ 
For now, `prettier` and `prettier/react` rules override `@m6web` rules.


```suggestion
      extends: ['airbnb', '@m6web', 'prettier', 'prettier/react'],
```